src/types.ts

```diff
@@ -108,6 +108,20 @@ export type SyncTeXRecordToTeX = {
     column: number
 }
 
+export type SyncTeXRecordToPDFAll = {
+    Page: number;
+    x: number;
+    y: number;
+    h: number;
+    v: number;
+    W: number;
+    H: number;
+}
```

Comment:
This type should be extended from `SyncTeXRecordToPDF`. If the original `indicator` has no use, either make it optional in parent and add corresponding logic, or ignore the value with proper comment annotation.